src/modules/Servicelicense/html_admin/mod_servicelicense_config.html.twig

```diff
@@ -85,7 +85,7 @@
             <li>{{ 'Take a look at default plugin at'|trans }} <strong>src/modules/Servicelicense/Plugin/Simple.php</strong></li>
             <li>{{ 'Create your plugin and place it to'|trans }} <strong>src/modules/Servicelicense/Plugin/PluginName.php</strong></li>
             <li>{{ 'FOSSBilling will automatically detect new plugin'|trans }}</li>
-            <li>{{ 'Read'|trans }} <a href="http://docs.fossbilling.org/en/latest/reference/service-license.html#license-plugin" target="_blank">{{ 'FOSSBilling documentation'|trans }}</a> {{ 'for more information.'|trans }}</li>
+            <li>{{ 'Read'|trans }} <a href="http://fossbilling.org/docs/en/latest/reference/service-license.html#license-plugin" target="_blank">{{ 'FOSSBilling documentation'|trans }}</a> {{ 'for more information.'|trans }}</li>
```

Comment:
I thought about changing this to https://fossbilling.org/docs/contribution-handbook/creating-an-extension, but as it is an empty file currently and I'm unsure if the license plugin documentation would be moved elsewhere, I ended up not doing it.

Reply:
Well, it looks like those are supposed specific instructions for creating an extension for the licensing module, which that doc wouldn't cover, even if it wasn't blank.
For now, that link is fine as-is, or we can point it to just the main docs page, either or is fine with me.

Reply:
Updated the broken links (except for the search one) to link to the main docs page.